2-mandatory/5-eligible-students.js

```diff
@@ -7,7 +7,11 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function getEligibleStudents() {}
+function getEligibleStudents(arr) {
+  let studentsAttendedOk = [];
+  arr.forEach((el) => (el[1] >= 8 ? studentsAttendedOk.push(el[0]) : null));
+  return studentsAttendedOk;
```

Comment:
There are a few ways of doing this. Using filter() and map() is a good alternative, but your method works well too.